v3/how-to-guides/01-basics/a-pallet-integration/index.mdx

```diff
@@ -5,6 +5,7 @@ keywords: basics, beginner, runtime
 version: '3.0'
 section: how to guides
 category: basics
+difficulty: 1
```

Comment:
Took this out originally to remove clutter

Reply:
So should this be removed from **all** HTGs?

Reply:
I went ahead and did this.